packages/camera/camera_android_camerax/lib/src/image_reader_rotated_preview.dart

```diff
@@ -157,9 +157,21 @@ final class _ImageReaderRotatedPreviewState
             sign: widget.facingSign,
           );
 
+          // If the camera is front facing, mirror the camera preview
+          // according to the current device orientation.
+          Widget cameraPreview = widget.child;
+          if (widget.facingSign == 1) {
```

Comment:
Why is == 1 the correct comparison here?

Reply:
Ah because `facingSign` maps to the `sign` in https://developer.android.com/media/camera/camera2/camera-preview#orientation_calculation, which is 1 for front cameras, -1 for back facing cameras. I'll add a comment to the code!